docs/guides/generator-attributes.md

```diff
@@ -43,17 +50,19 @@ struct __attribute__((eosio_table)) testtable {
 
 typedef eosio::multi_index<"tablename"_n, testtable> testtable_t;
 ```
-If you don't want to use the multi-index you can explicitly specify the name in the attribute ```c++ [[eosio::table("<valid action name>")]]```.
+If you don't want to use the multi-index you can explicitly specify the name in the attribute `c++ [[eosio::table("<valid action name>")]]`.
```

Comment:
@afalaleev `eosio::table` is deprecated. Actual attributes (`eosio::order`, etc.) not described. For what such docs?